components/portal/messages/partials/notifications-list-item.html

```diff
@@ -19,23 +19,35 @@
 
 -->
 <!-- NOTIFICATION CONTENT -->
-<a class="md-list-item-text"
+<div class="md-list-item-text"
    ng-href="{{ notification.actionButton.url }}"
-   target="_blank"
-   rel="noopener noreferrer"
-   aria-label="{{ notification.actionButton.label }}"
    layout="row"
    layout-align="start center"
    layout-xs="column"
    layout-align-xs="center start">
-  <span><md-icon ng-if="notification.priority == 'high'" aria-label="high priority:">priority_high</md-icon> {{ notification.title }}</span>
-  <div ng-if="notification.moreInfoButton" class="notification-buttons" layout-xs="column" layout-align-xs="center start">
+  <!-- Notification text -->
+  <div layout="column" layout-align="center start" flex-gt-xs="70">
+    <div class="notification__priority-message" ng-if="notification.priority === 'high'">
+      <md-icon class="md-warn" aria-label="high priority:">priority_high</md-icon>
+      <span>High priority</span>
+    </div>
+    <div>{{ notification.title }}</div>
+  </div>
+  <!-- Notification buttons -->
+  <div ng-if="notification.moreInfoButton || notification.actionButton" class="notification-buttons" flex-gt-xs="30" layout="row" layout-align="end center">
+    <md-button class="md-raised md-accent"
+               ng-href="{{ notification.actionButton.url }}"
```

Comment:
:sparkles: https://material.angularjs.org/latest/api/directive/mdButton

> If you supply a `href` or `ng-href` attribute, it will become an `<a>` element. Otherwise, it will become a `<button>` element.

Thats pretty cool